core/pom.xml

```diff
@@ -267,6 +267,25 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
```

Comment:
shouldn't we add those for `mapper-processor` and `mapper-runtime` as well and adapt 
`PlatformInfoFinder#getRuntimeInfo` to take into account two new files?

Reply:
done